social_core/utils.py

```diff
@@ -2,11 +2,15 @@
 import sys
 import time
 import unicodedata
-import collections
 import functools
 import hmac
 import logging
 
+try:
+    from collections.abc import Callable
+except ImportError:
+    from collections import Callable
```

Comment:
Could use `six.moves.collections.abc.Callable` on both versions.

Reply:
Although actually, the `callable()` builtin would do just as well in these cases.